<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitTextCn</class>
 <widget class="QMainWindow" name="SplitTextCn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>中文分词</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_chooseOriginalFiles">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>选择分词文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chooseResultFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>选择结果文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chooseStopDict">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>选择停用词词典</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chooseSelfDict">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>选择自定义词典</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_run">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 155, 255);</string>
    </property>
    <property name="text">
     <string>开始分词</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_OriginalFiles">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ResultFile">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_StopDict">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SelfDict">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_full">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>全模式</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_precise">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>精确模式</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_searchengine">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索引擎模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
